api/tests/mocha/services/users.spec.js

```diff
@@ -587,7 +587,7 @@ describe('Users service', () => {
         password: 'short'
       }).catch(err => {
         chai.expect(err.code).to.equal(400);
-        chai.expect(err.message).to.equal('The password must be at least 8 characters long.');
+        chai.expect(err,{ message:'The password must be at least 8 characters long.', key: 'password.length.minimum', field: 'minimum', parameter: '8'});
```

Comment:
`chai.expect` doesn't accept 2 parameters. 
https://www.chaijs.com/api/bdd/

You'd have to use it like `chai.expect(err.message).to.equal(...)` or `chai.expect(err.translation_key).to.equal(...)`